Intents.intentdefinition

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>INEnums</key>
+	<array/>
+	<key>INIntentDefinitionModelVersion</key>
+	<string>1.2</string>
+	<key>INIntentDefinitionNamespace</key>
+	<string>1AigIl</string>
+	<key>INIntentDefinitionSystemVersion</key>
+	<string>23F79</string>
+	<key>INIntentDefinitionToolsBuildVersion</key>
+	<string>15F31d</string>
+	<key>INIntentDefinitionToolsVersion</key>
+	<string>15.4</string>
+	<key>INIntents</key>
+	<array>
+		<dict>
+			<key>INIntentCategory</key>
+			<string>generic</string>
+			<key>INIntentConfigurable</key>
+			<true/>
+			<key>INIntentDescription</key>
+			<string>starts tracking with gps</string>
```

Comment:
I don't think this is a correct description. It doesn't necessarily use GPS. Also need proper letter cases